woocommerce-transbank/libwebpay/Telemetry/PluginVersion.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Transbank\Telemetry;
+
+class PluginVersion
+{
+    protected $soapUri = 'http://www.cumbregroup.com/tbk-webservice/PluginVersion.php?wsdl';
+    protected $client;
+    
+    const ENV_INTEGRATION = 'INTEGRACION';
+    const ENV_PRODUCTION = 'PRODUCCION';
+    const PRODUCT_WEBPAY = 1;
+    
+    const ECOMMERCE_WOOCOMMERCE = 1;
+    const ECOMMERCE_PRESTASHOP = 2;
+    const ECOMMERCE_MAGENTO2 = 3;
+    const ECOMMERCE_VIRTUEMART = 4;
+    const ECOMMERCE_OPENCART = 5;
+    const ECOMMERCE_SDK = 6;
```

Comment:
Why is included the other ecommerce? I think that we should remove the unused constants.

Reply:
I want to use the exact same class on other plugins, and eventually export it as a package. This class would be like the 'Internal Telemetry SDK'. At least, that was what i was thinking about when I put all the constants there. Do I remove them anyway?

Reply:
cc @a1b4

Reply:
ok I understand, but I need that you create a Jira card for the creation of the package.